service/src/main/java/uk/gov/hmcts/reform/fpl/controllers/OthersController.java

```diff
@@ -49,7 +50,12 @@ public AboutToStartOrSubmitCallbackResponse handleAboutToSubmit(@RequestBody Cal
 
         List<Element<Other>> others = confidentialService.removeConfidentialDetails(allOthers);
 
-        caseDetails.getData().put("others", Others.from(others));
+        Others othersList = Others.from(others);
+        if (isNull(othersList)) {
+            caseDetails.getData().remove("others");
```

Comment:
I had to remove "others" fields as some case data already have "others" value empty